homeassistant/components/shelly/sensor.py

```diff
@@ -122,12 +125,31 @@
     ("sensor", "tilt"): BlockAttributeDescription(name="tilt", unit=DEGREE),
 }
 
+REST_SENSORS = {
+    ("rssi"): RestAttributeDescription(
+        name="Rssi",
+        unit="dB",
+        device_class=sensor.DEVICE_CLASS_SIGNAL_STRENGTH,
+        default_enabled=False,
+        path="wifi_sta/rssi",
+    ),
+    ("uptime"): RestAttributeDescription(
+        name="Uptime",
+        device_class=sensor.DEVICE_CLASS_TIMESTAMP,
+        icon="mdi:clock",
```

Comment:
Don't use `icon` if you're using `device_class`.

Reply:
TIMESTAMP class has a generic eye as icon, this is why in this special case I added one.

Reply:
Yes, I know, but the rule is that if we're using `device_class`, we're not using an `icon`.

Reply:
Done :-(